.pfnci/run.sh

```diff
@@ -187,11 +187,55 @@ test_py27and35() {
   exit $((py27_test_status || py35_test_status))
 }
 
+# test_chainermn is a test function for chainermn
+test_chainermn() {
+  export PYENV_VERSION=""
+  . /root/.bash_profile
+
+  pyenv shell "2.7.16-cupy-${CUPY_VERSION}"
```

Comment:
Can we use list here?
And we need to aggregate the return values

```
ret=0
TEST_CASES="2.7.16-cupy-${CUPY_VERSION} 3.5.7-cupy-${CUPY_VERSION} 3.7.3-cupy-${CUPY_VERSION}"
for TEST in $TEST_CASES
do
     pyenv shell $TEST
     test_chainermn_sub
     tmp_ret=$?
     ret=$(( ret || tmp_ret))
done
exit $ret
```

Reply:
Fixed